rulesets.go

```diff
@@ -734,19 +730,16 @@ type UpdateRulesetParams struct {
 }
 
 type UpdateEntrypointRulesetParams struct {
-	Name        string        `json:"name,omitempty"`
```

Comment:
This technically can be specified but has some slightly inconsistent behaviour from the API if it is. If you specify it and the entry point ruleset didn't exist before, then this name will be used. If the entry point ruleset already exists though, this name will be ignored (and the ruleset will not be renamed).

Omitting the name always works consistently and will cause the API to set it to "default" if an entry point ruleset does not already exist. This is why I think it's probably best to not allow users to set the name explicitly to avoid confusion, but we can keep it here if that would be too much of a breaking change. What do you think @jacobbednarz?

Reply:
works for me!